style.css

```diff
@@ -237,6 +241,51 @@ img {
   border-right: 1px solid black;
 }
 
+.form-floating textarea {
+  overflow: hidden;
+}
+
+.sign-up-title-form {
+  text-align: center;
+}
+
+.sign-up-title {
+  color: #caad1c;
+}
+
+.sign-up-instructions {
+  color: #0d518c;
+}
+
+#floatingName {
+  width: 275px;
```

Comment:
I see you've added custom width's to all of the fields and it's more unique! I would love to know how you came up with the exact width's😊 If you don't think they all look bad at the default 100% width, it works! Attached below is a screenshot of the fields with no certain width applied to them to make for an easier explanation!
![Screen Shot 2021-05-12 at 11 18 22 PM](https://user-images.githubusercontent.com/35432129/118072688-5c3ed080-b378-11eb-9055-9d78c2c081bc.png)

Reply:
Yes, I did have the initial width of 100% in the beginning but for some reason it felt like the input boxes were taking up too much of the modal so I thought if I shortened a few based on what was being asked that maybe it would look better. What do you think?

Reply:
Lol, okay! The 100% width didn't bother me at all and did look like a common sign-up form. But this is your canvas so if you'd prefer the widths you applied, I'm all for it!

Reply:
Yea after looking at them for a while I think the 100% might be the best option. Thanks for that catch